quirks/websites-with-shared-credential-backends.json

```diff
@@ -311,6 +311,10 @@
         "ebay.pl",
         "ebay.vn"
     ],
+    [
+        "epicgames.com",
+        "unrealengine.com"
+    ],
```

Comment:
This file is generated file from `shared-credentials.json` and `shared-credentials-historical.json` using `convert-shared-credential-to-legacy-format.rb` and is not meant to be manually edited.

Can you please move these domains to `shared-credentials-historical.json` and regenerate this file by running `convert-shared-credential-to-legacy-format.rb`?

You can find more info about the shared credentials in the [README](https://github.com/apple/password-manager-resources#shared-credentials) and [CONTRIBUTING](https://github.com/apple/password-manager-resources/blob/main/CONTRIBUTING.md#contributing-a-shared-credentials-entry).